src/main/resources/i18n/messages.properties

```diff
@@ -3023,3 +3023,7 @@ AnalysisState.ERROR=Error
 # InputWithOptions                                                                        #
 #=========================================================================================#
 InputWithOptions.required={0} is required to run this pipeline. Please select one of the options.
+
+IridaOAuthErrorHandler.unauthorized=User is unauthorized for this service
+IridaOAuthErrorHandler.forbidden=Access denied. User is forbidden from accessing this service
```

Comment:
Can you update this error message to `Access denied. User is forbidden from accessing this resource`